src/test/kotlin/eu/theblob42/idea/whichkey/config/PopupConfigCodeTest.kt

```diff
@@ -0,0 +1,4 @@
+package eu.theblob42.idea.whichkey.config
+
+import org.junit.jupiter.api.Assertions.*
+ class PopupConfigCodeTest
```

Comment:
Did you add this new file by accident to this PR?

Reply:
Oops, yes